.github/workflows/ci_tests.yaml

```diff
@@ -82,18 +82,66 @@ jobs:
 
     - name: E2E Tests
       run: |
-        make deploy-ci NAMESPACE=mygatekeeper IMG=localhost:5000/gatekeeper-operator:$GITHUB_SHA
-        kubectl -n mygatekeeper wait deployment/gatekeeper-operator-controller --for condition=Available --timeout=90s
-        kubectl -n mygatekeeper logs deployment/gatekeeper-operator-controller -c manager -f > operator.log &
-        make test-e2e NAMESPACE=mygatekeeper
-        kubectl delete --wait namespace mygatekeeper
+        make deploy-ci NAMESPACE=gatekeeper-system IMG=localhost:5000/gatekeeper-operator:$GITHUB_SHA
+        kubectl -n gatekeeper-system wait deployment/gatekeeper-operator-controller --for condition=Available --timeout=90s
+        kubectl -n gatekeeper-system logs deployment/gatekeeper-operator-controller -c manager -f > operator.log &
+        make test-e2e NAMESPACE=gatekeeper-system
```

Comment:
Why the changes to `gatekeeper-system`? I think that's the default? I might prefer we use `mygatekeeper` for all or some tests so that we know that deploying to something other than the default namespace still works?

Reply:
Oops. This is a comment for the other PR. I'll move it over.

> Why the changes to `gatekeeper-system`? I think that's the default? I might prefer we use `mygatekeeper` for all or some tests so that we know that deploying to something other than the default namespace still works?

Oops. This is a comment for the other PR. I'll move it over.